app/src/main/java/com/waseefakhtar/doseapp/feature/addmedication/AddMedicationRoute.kt

```diff
@@ -457,7 +454,7 @@ private fun canSelectMoreTimesOfDay(selectionCount: Int, numberOfDosage: Int): B
     return selectionCount < numberOfDosage
 }
 
-private fun showMaxSelectionSnackbar(
+private fun showMaxSelectionSnackBar(
```

Comment:
Let's keep this `showMaxSelectionSnackbar`.

Reply:
I have resolved this one